jazon-core/src/main/java/com/zendesk/jazon/actual/ActualJsonArray.java

```diff
@@ -27,4 +27,15 @@
     public MatchResult accept(JsonExpectation expectation, String path) {
         return expectation.match(this, path);
     }
+
+    @Override
+    public String toString() {
+        return "[" + String.join(", ", strings(list)) + "]";
+    }
+
+    private static List<String> strings(List<Actual> actuals) {
+        return actuals.stream()
+                .map(Objects::toString)
+                .collect(toList());
+    }
```

Comment:
Minor: I just wonder if that needs to be static. Maybe this would be enough?
```java
    private List<String> strings() {
        return this.list.stream()  # or this.list(), but we don't modify it in-place here
                .map(Objects::toString)
                .collect(toList());
    }
```

You handled it similarly in `ActualJsonObject`'s `fields()` method.

Reply:
Oh good catch :)